services/gateway/apis/http-version3/swagger/definitions/networkStatus.json

```
@@ -94,6 +94,10 @@
                     },
                     "registeredModules": {
                         "type": "array",
+                        "items": {
+                            "type": "string",
+                            "description": "Array of all the current registered modules"
```

Please add `.` at the end of all the descriptions. And, please update `Array` to `List`.
```suggestion
                            "description": "List of all the current registered modules."
```